staging_service/autodetect/Mappings.py

```diff
@@ -91,10 +91,12 @@ def _add_gzip(extension_list):
     EXCEL: ["xls", "xlsx"],
     ZIP: ["zip", "tar", "tgz", "tar.gz", "7z", "gz", "gzip", "rar"],
     SBML: ["smbl"],
-    # Custom File Types?
-    MEDIA: ["tsv", "xls", "xlsx"],
-    PHENOTYPE: ["tsv"],
-    ESCHER: ["json"],
-    ANNOTATIONS: ["tsv"],
-    FBA: ["tsv", "xls", "xlsx", "smbl"],
 }
+
+extension_to_type_mapping = {}
+for type_, extensions in type_to_extension_mapping.items():
```

Comment:
Maybe "type_to_extension_mapping" needs to be renamed to "file_format_to_extension_mapping"

Reply:
Done